src/Benchmarks/Startup.cs

```diff
@@ -108,6 +109,78 @@ public void ConfigureServices(IServiceCollection services)
 
                     break;
 #endif
+
+                case DatabaseServer.Sqlite:
+                    using (var connection = new SqliteConnection(appSettings.ConnectionString))
+                    {
+                        if (!File.Exists(connection.DataSource))
+                        {
+                            connection.Open();
+
+                            var command = connection.CreateCommand();
+                            command.CommandText =
+                            @"
+                                PRAGMA journal_mode = 'wal';
```

Comment:
That should not be the default ... maybe a command line argument, or ENV, so we can decide to test different values. I think the defaults should be whatever sqlite default is if we don't set it.

Reply:
If taken from dotnet config we then have the choice for ENV or command line, there are other argument in this project, should be easy to copy.

Reply:
I was debating whether to do that or not. Will update.

Reply:
That's something I definitely want to test, whether it's better or not. And we have different kinds of payload in this app, reads, inserts, updates ... so that should give us more real feedback.

Reply:
Note, WAL is the default for databases created by EF Core.

Reply:
Yeah, the official SQLite guidance says that it's better for concurrent access to the database where there are some writes. But it'll be nice to finally prove (or disprove) that.